app/bundles/PluginBundle/Entity/Plugin.php

```diff
@@ -21,6 +21,8 @@
  */
 class Plugin extends CommonEntity
 {
+    const DESCRIPTION_DELIMITER = "\n---\n";
```

Comment:
For windows support just add, then will works properly

` const DESCRIPTION_DELIMITER = "\r\n---\r\n";`

![image](https://user-images.githubusercontent.com/462477/37594949-2c29e3d0-2b77-11e8-9a94-16083048ffbd.png)